index.bs

```diff
@@ -1234,7 +1231,25 @@ intentionally mirror settable attributes and methods of the
     if a cue ID is invalid (removing an un-added ID or adding an ID twice, for example),
     the receiver may reject the text track change.
 
-Issue: Add a table for whether it's required and what the default is.
+<div class="note">
+<table>
+<tr><th>Field</th>                  <th>Default value</th>                              <th>Receiver support</th></tr>
+<tr><td>source</td>                 <td>`urls` in [=remote-playback-start-request=]</td><td>Required</td></tr>
+<tr><td>preload</td>                <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>loop</td>                   <td>False</td>                                      <td>Required</td></tr>
+<tr><td>paused</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>muted</td>                  <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>volume</td>                 <td>Decided by the receiver</td>                    <td>Required</td></tr>
+<tr><td>seek</td>                   <td>-</td>                                          <td>Required</td></tr>
+<tr><td>fast-seek</td>              <td>-</td>                                          <td>Required</td></tr>
+<tr><td>playback-rate</td>          <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>poster</td>                 <td>Decided by the receiver</td>                    <td>Not required</td></tr>
+<tr><td>enabled-audio-track-ids</td><td>-</td>                                          <td>Required</td></tr>
```

Comment:
These IDs are for the *controls*, not the *state*.  There are 2 different messages.   Maybe we should have a table for both, but the issue/TODO was for the state table, so we need at least that.